website/docs/cloud-docs/no-code-provisioning/module-design.mdx

```diff
@@ -46,6 +47,15 @@ For example, you could build modules that satisfy the following well-scoped use
 - Deploying all resources needed for a three-tier web application
 - Deploying a database with constraints on resource allocation and deployment region
 
+### Adjust for New Module Versions
+No-code provisioning desgination is now pinned to a selected version of a module. You must select the version at the time of enabling the module. If no selection is made, it will default to the latest version at the time of enabling. This ensures that you can define variable input options for the intended version and that no-code users provision the most appropriate version. When a newer version of the module becomes avialable, you may want to update the pinned no-code version to the latest.
+
 ### Provide Variable Defaults When Possible
 
-The no-code provisioning workflow prompts users to set values for the module's input variables before creating the new workspace and deploying resources. We recommend setting reasonable defaults when possible to reduce the effort and expertise needed to deploy the module. Remember that the workspace can also access variable values set through global variable sets in your organization.
+The no-code provisioning workflow prompts users to set values for the module version's input variables before creating the new workspace and deploying resources. We recommend setting reasonable defaults when possible to reduce the effort and expertise needed to deploy the module. Remember that the workspace can also access variable values set through global variable sets in your organization.
+
+### Define Dropdown Options for Variables without Defaults
+
+For variables without defaults, you may still want to limit what the no-code user can input. Define these input options at the time of enabling the module for no-code provisioning. You can do this either from the Terraform Cloud UI, via the [No-Code Provisioing API](), or via the [TFE provider](). The latter method is recommended for those that want to track these variable input options through a code approval process, keeping the configuration as code.
```

Comment:
```suggestion
For variables without defaults, you may still want to limit what the no-code user can input. Define these input options at the time of enabling the module for no-code provisioning. You can do this either from the Terraform Cloud UI, via the [No-Code Provisioing API](https://developer.hashicorp.com/terraform/cloud-docs/api-docs/no-code-provisioning), or via the [TFE provider](https://registry.terraform.io/providers/hashicorp/tfe/latest/docs/resources/no_code_module). The latter method is recommended for those that want to track these variable input options through a code approval process, keeping the configuration as code.
```

Reply:
@opslivia (Un-resolving this thread too) Did you see my suggestion above?

Reply:
(Closing because I resuggested these in a new review)